device/mellanox/x86_64-mlnx_msn4600c-r0/ACS-MSN4600C/sai_4600C.xml

```diff
@@ -467,4 +483,4 @@
             </port-info>
         </ports-list>
     </platform_info>
-</root>
```

Comment:
Can we remove this extra change?

Reply:
I see that the </root> was removed and added right after. I don't know why it was removed and re-added, the changes derived from a tool